rmf_site_format/src/sdf.rs

```diff
@@ -23,12 +23,19 @@ use once_cell::sync::Lazy;
 use sdformat_rs::*;
 use std::collections::BTreeMap;
 use thiserror::Error;
+use bevy::{prelude::*, utils::tracing};
+use tracing::error;
 
 const DEFAULT_CABIN_MASS: f64 = 1200.0;
 
 static WORLD_TEMPLATE: Lazy<SdfRoot> = Lazy::new(|| {
-    yaserde::de::from_str(include_str!("templates/gz_world.sdf"))
-        .expect("Failed deserializing template")
+    match yaserde::de::from_str(include_str!("templates/gz_world.sdf")) {
```

Comment:
Since this can have cascading ramifications if it fails, I think the right thing to do here is

```rust
static WORLD_TEMPLATE: Lazy<Result<SdfRoot, String>> = ...
```

Then in `Site::to_sdf` where this gets used, we return an `Err` result if `WORLD_TEMPLATE` has an `Err` value.

We can add a variant to `SdfConversionError` like `CorruptedWorldTemplate` to cover this error case.